lib/components/Charts/utils/bar.ts

```diff
@@ -0,0 +1,5 @@
+import { ChartItem } from "./types"
+
+export function prepareData<Keys extends string>(data: ChartItem<Keys>[]) {
+  return data.map((item) => ({ x: item.label, ...item.values }))
+}
```

Comment:
Could we perhaps use this function on the `LineChart` component as well so it's not duplicated?